### DISCUSSION TITLE
Commenting on old review publishes the current draft & one extra unwanted

### DISCUSSION THREAD

**Discussion by janbrasna** [community/community#7811] [Pull Requests]
I write a couple of review comments, starting a review… intending to publish it later once I cover all the changes to finish it w/ a summary post at some point later.

Now I see an older review thread that needs updating, so I comment on an older review.

After submitting this comment to an existing review, what happens is:
1. The comment for older review gets posted ✅ 
2. All the previous unfinished draft review comments get published as single comments w/o any action or confirmation whatsoever 😮 
3. The comment to the older thread becomes also published as a new review, after the above drafts, as a duplicate detached from the original thread 🤬

This can't be the expected behaviour, right? Could you please anyone try the above workflow, if that happens for anyone else, or something's broken only for me? Thanks~

**Comment by zzJinux** [comment]
I have been experiencing the same issue from GHES 3.4.3 after which I have found this discussion. It's unfortunate that it is not spotlighted as much I want.

The most critical flaw I think is that it breaks the mental model of posting comments: one block for one comment. It's really awkward that two identical comment blocks are created, even for hyperlinks! (Try "Copy link" from the three-dots menu. Both comment block gives you exactly the same url)
